### ISSUE TITLE
Can not load .py files on the file system

### ISSUE THREAD

**Issue by vaibhavgarg1982** [pyscript/pyscript#209]
Attempting to access any .py file from within the HTML page fails. The error shown in console is as under: 

`Access to fetch at 'file:///C:/data.py' from origin 'null' has been blocked by CORS policy: Cross origin requests are only supported for protocol schemes: http, data, chrome-extension, edge, https, chrome-untrusted.`

For reference, I am running this on Windows 11, over Edge-Chromium. Also, in lining the code works as expected.

**Comment by work-ruchir** [comment]
Try running your page on a server (live server, http server). Live server can directly be installed from VSCODE marketplace.

**Comment by antocuni** [comment]
Duplicate of #179

**Comment by marimeireles** [comment]
Tracked on #756
